getting-started/clustering-installation.md

```diff
@@ -1,9 +1,17 @@
 # Installation
 
-TimescaleDB multinode and distributed capabilities are currently in Beta. To learn more,
-you can read the announcement [blog post][blog-post].
+TimescaleDB multinode and distributed capabilities are currently in Beta. If you are interested
+in joining the Beta program, please [contact us][contact].
 
-If you are interested in joining the Beta program, please [contact us][contact].
+To learn more about this capability, you can read the following:
+
+- Blog: [Announcing the Distributed Version of TimescaleDB][blog-post]
+- Documentation: [Using Distributed Hypertables][scale-out]
```

Comment:
Creating distributed hypertables should be the `/getting-started/scale-out` link?

Reply:
Made a slight wording change (in subsequent commit). Think that helps clarify a bit more.